tsconfig.json

```diff
@@ -1,34 +1,16 @@
 {
   "compilerOptions": {
-    "module": "esnext",
```

Comment:
You removed or changed options to older values, such as from `esnext` to `commonjs`. Could you start from the original and change only a minimum number of options?

Reply:
Only removed:
    "types": ["react"] and "strict": true,